docs/tutorial_bracket_strategy.md

```diff
@@ -192,20 +200,24 @@ In case you don't remember them, you would have to run the following command:
 npx truffle exec scripts/get_deployed_brackets.js --masterSafe=$MASTER_SAFE --network=$NETWORK_NAME
 ```
 
-which will display all your previously written brackets.
+which will display all your previously written brackets and generate a csv listing the brackets with more information.
 
 By running the withdraw request script, you have generated a transaction within your Gnosis-safe interface. Execute this transaction and wait - maximal 5 minutes after the transaction gets mined - until the gnosis protocol closes its batch.
 In the next batch you will be able to withdraw your funds.
 
 ### Withdrawing funds
 
-```
+The actual withdraw transaction transferring the tokens back into the MASTER SAFE from the Gnosis Protocol can be initiated with the following command.
+
+```ssh
 npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma separated brackets]  --tokenIds=[indices] --withdraw --network=$MASTER_SAFE
 ```
 
+The parameters are basically the same as in the requesting withdraw script, besides the new flag `--withdraw` instead of `--withdrawRequest`.
+
 ## Even more secure setup with more owners
```

Comment:
Maybe a section on: tracking my orders / tracking my portfolio?

Reply:
Good point. Let's do this once the code is ready for that...

Reply:
```suggestion
## Securing your setup with more owners
```